src/components/Analisis.jsx

```diff
@@ -27,7 +27,6 @@ export default function Analisis({ min, max, total, tipos, componentes }) {
       </div>
 
       <div className={styles.analisisDatos}>
-
```

Comment:
was this required by the linter ?

Reply:
Yeah, it's the thing that's breaking in the latest PRs